QUEST_JOBSTEP.tsv

```diff
@@ -753,42 +753,42 @@ QUEST_JOBSTEP_20150317_000752	The disciples came to Lydia Schaffen. Afterwards,
+QUEST_JOBSTEP_20150317_000767	$'Of course, a game to see who can shoot better with a bow would not be that interesting. Therefore, let's play a game where you fire an arrow with your bow at the target, and then, I will try to catch and ride on the arrow before the arrow reaches the target,' said Rumpelstiltskin. {nl}Lydia replied back, questioning the proposal, 'Ride on the arrow?'{nl}'That is correct,' averred Rumpelstiltskin.{nl}
+QUEST_JOBSTEP_20150317_000768	$Upon answering, Rumpelstiltskin started to shrink his body.{nl} Once he had reduced himself to the size of a child's fist, Rumpelstiltskin spoke, 'Well, this size will allow me to ride on your arrow, right?'{nl}Lydia Schaffen thought for a bit,{nl}'That's a problem for the match. If you touch the arrow and cause it to fall to the ground without it ever hitting the target, then wouldn't that still be a loss for me regardless of whether you rode the arrow or not?'{nl}Lydia answered Rumpelstiltskin with these words.{nl}
+QUEST_JOBSTEP_20150317_000769	$'Then how about this. I win if I can ride your arrow until it reaches the target,' answered Rumpelstiltskin.{nl} 'And if your arrow falls to the ground,' he continued, 'then I will lose as well.'{nl}Rumpelstiltskin added, 'The game shall have 3 rounds, best 2 out of 3.'{nl}
+QUEST_JOBSTEP_20150317_000770	$After Rumpelstiltskin's seemingly honest answer, Lydia Schaffen replied, 'Alright. One more question though. In a place like this, what sort of targets will I shoot at? I mean you won't be used to riding the arrows until round 2 or round 3, so the targets for each round should be called in advance. Don't you agree?'{nl}'Alright, let's do that,' agreed Rumpelstiltskin. {nl}And after agreeing on compensation upon defeat as well, the game between two people, err, one person and one demon began.{nl}
+QUEST_JOBSTEP_20150317_000771	$Lydia Schaffen weighed her arrow, then called out the first target,{nl}'The tawny tree over there!'{nl}Together with those words in advance she fired her bowstring, the arrow's frightful speed sliced through the air.{nl}
+QUEST_JOBSTEP_20150317_000772	$Rumpelstiltskin moved nimbly to chase the flying arrow, and after he got on, the arrow impaled the tree. As the arrow continued to oscillate, he danced on the arrow and sang an unknown tune.{nl}Lydia Schaffen again gauged her arrows, then she called,{nl}'All of the pinecones from the pine tree on my front left side!'{nl}Upon calling the target, Lydia Schaffen unveiled her godlike archery skills. 
+QUEST_JOBSTEP_20150317_000773	$Her arrows poured faster than heavy rain during a hot summer day, as if the white tailed arrows were fired at the same time in the same direction. {nl}
```

Comment:
"Her arrows poured faster" -> "Her arrows poured out faster"

"white tailed" -> "white-tailed"

Are you sure "heavy rain during a hot summer day" is the right metaphor?